config/manifests/bases/machine-deletion-remediation.clusterserviceversion.yaml

```diff
@@ -40,10 +40,24 @@ spec:
       - displayName: Template
         path: template
       version: v1alpha1
-  description: This operator follows the annotation on the Node to the associated
-    Machine object, confirms that it has an owning controller (e.g. MachineSetController),
-    that will recreate the Machine once deleted, and then proceeds with the deletion
-    of the Machine.
+  description: |
+    Machine Deletion Remediation (MDR) is a remediator designed to reprovision unhealthy
+    nodes using the Machine API. MDR can be used together with NodeHealthCheck (NHC),
+    which detects the unhealthy Node and creates the Custom Resource MDR needs to remediate it.
```

Comment:
`creates the Custom Resource MDR needs to remediate it.`
is a bit confusing for me.  I assume that by `it` you refer to the node but it took me couple of reads to underdtand that (initially I thought you wrongly refer to the CR).

Reply:
I agree, it's too obscure, I'll rephrase it. Maybe just `and creates the Custom Resource for MDR` would do it